cmd/dep/integration_test.go

```diff
@@ -35,8 +35,6 @@ func TestIntegration(t *testing.T) {
 			parse := strings.Split(path, string(filepath.Separator))
 			testName := strings.Join(parse[2:len(parse)-1], "/")
 			t.Run(testName, func(t *testing.T) {
-				t.Parallel()
```

Comment:
Yeah this was an attempt to mitigate the race popping up on the integration test buffers. Definitely trying to keep this in.